src/components/Matches/index.js

```diff
@@ -0,0 +1,3 @@
+import Matches from './Matches';
```

Comment:
if you want to, you could just rename your Matches.jsx to index.jsx to avoid the reimport/export step.